pom.xml

```diff
@@ -71,7 +71,7 @@
   <properties>
     <revision>2.81</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.222.4</jenkins.version>
+    <jenkins.version>2.332</jenkins.version>
```

Comment:
No `2.332.1` available yet

Reply:
adding a `TODO` to change it to a LTS when it's out?

Reply:
Done